src/cec/command-map.ts

```diff
@@ -0,0 +1,237 @@
+/**
+ * @packageDocumentation
+ * Command Map Module. Maps CEC event codes to Jellyfin commands.
+ */
+
+/**
+ * Parses CEC event code and converts it into a Jellyfin command.
+ * @param code 2 character code.
+ */
+export function parseCmd(code: string): string | undefined {
+    switch (code.toUpperCase()) {
```

Comment:
This would be a perfect use or a lookup table at some point.

Reply:
Yeah tbd currently it's just conversion to ts